kairos-cli/bin/crypto/private_key.rs

```diff
@@ -0,0 +1,18 @@
+use casper_types::file_utils::read_file;
+
+use crate::crypto::error::CryptoError;
+
+pub struct CasperPrivateKey(pub casper_types::SecretKey);
+
+impl CasperPrivateKey {
+    pub fn from_file(file_path: &str) -> Result<Self, CryptoError> {
+        let data = read_file(file_path).map_err(|_e| CryptoError::KeyLoad)?;
+        let secret_key =
+            casper_types::SecretKey::from_pem(data).map_err(|_e| CryptoError::FailedToParseKey)?;
+        Ok(Self(secret_key))
+    }
```

Comment:
Change the arguments to match their signature. It use `&Path` via `AsRef`, using `&Path` directly is fine as well.

```suggestion
    pub fn from_file<P: AsRef<Path>>(file_path: P) -> Result<Self, ErrorExt> {
        casper_types::SecretKey::from_file(file_path).map(Self)
    }
```

Reply:
Thanks, improved in https://github.com/cspr-rad/kairos/pull/17/commits/b1a78b47dfce7f834e6d5f05e824b217beffa53f.